.appveyor.yml

```diff
@@ -1,8 +1,5 @@
 environment:
   matrix:
-  - nodejs_version: 4
-  - nodejs_version: 6
-  - nodejs_version: 7
```

Comment:
why would node 4 6 and 7 deleted? should node 9 be added here as well?

Reply:
I am intentionally reducing the _Windows_ footprint to just the LTS release because AppVeyor queues are getting very high. I think it is reasonable to test on all versions on Linux, and just LTS on Windows.